5Total.ipynb

```diff
@@ -0,0 +1,419 @@
+{
```

Comment:
<p>Very good idea to do it in python first. This is probably also how you do it in reality. You develop in python and then optimize in c++</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ubsuny/CP1_PythonCpp/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>